framework/src/geomsearch/SlaveNeighborhoodThread.C

```diff
@@ -70,40 +62,30 @@ SlaveNeighborhoodThread::operator()(const NodeIdRange & range)
 {
   processor_id_type processor_id = _mesh.processor_id();
 
-  for (const auto & node_id : range)
-  {
-    const Node & node = *_mesh.nodePtr(node_id);
-
-    std::priority_queue<std::pair<unsigned int, Real>,
-                        std::vector<std::pair<unsigned int, Real>>,
-                        ComparePair>
-        neighbors;
+  unsigned int patch_size = std::min(
+      _patch_size, static_cast<unsigned int>(_trial_master_nodes.size()));
 
-    unsigned int n_master_nodes = _trial_master_nodes.size();
+  std::vector<size_t> ret_index(patch_size);
```

Comment:
std::size_t instead of size_t. In C++ it's in a namespace, you are picking up the C version.